pinot-controller/src/main/java/com/linkedin/pinot/controller/api/resources/PinotSegmentUploadRestletResource.java

```diff
@@ -407,6 +407,9 @@ private void uploadSegment(File indexDir, SegmentMetadata segmentMetadata, File
     TableConfig offlineTableConfig =
         ZKMetadataProvider.getOfflineTableConfig(_pinotHelixResourceManager.getPropertyStore(), offlineTableName);
 
+    LOGGER.info("Starting upload for segment {}, tempTarredSegmentFile {}", segmentName,
```

Comment:
may want to add a comment here (after verification) that the time difference between this log and the one that logs the headers gives us the time taken to untar the segment. Useful for debugging if storage is slow.